docs/index.md

```diff
@@ -71,6 +71,34 @@ containerd will come with a default implementation for the various components.
 These defaults will be chosen by the maintainers of the project and should not change unless better tech for that component comes out.
 Additional implementations will not be accepted into the core repository and should be developed in a separate repository not maintained by the containerd maintainers.
 
+### Build it
+You can build `containerd` via Docker container. You can build an image from
+this Dockerfile:
+
```

Comment:
With this chapter, I am trying to build a local version of `containerd`. I probably didn't get your feedback @AkihiroSuda I can `RUN make` inside the container but I probably need to add/copy the code into the container.